libraries/ms-drivers/inc/ltc6811.h

```diff
@@ -4,10 +4,18 @@
 #include "ltc_afe.h"
 
 // used internally by the LTC AFE driver
-
 #define LTC6811_CELLS_IN_REG 3
 #define LTC6811_GPIOS_IN_REG 3
 
+// used for the external mux (ADG731) connected to the AFE
+#define AUX_ADG731_NUM_PINS 32
+
+// Size of command code + PEC
+#define LTC6811_CMD_SIZE 4
+
+// 3 bytes are required to send 24 clock cycles with our SPI driver for the STCOMM command
+#define LTC6811_24_CLOCK_CYCLES_STCOMM_BYTES 3
```

Comment:
Current name is fine but I feel like something like `_COMM_REG_SIZE` or `_NUM_COMM_REG_BYTES`  woulda been better since that's what it is (and then in the comment say what it's used for)

Reply:
This isn't the COMM reg's size though? This is just for the NULL bytes when I send the clock cycles

Reply:
You use it to send 3 NULL bytes but the reason it's 3 is because the COMM register sends 3 bytes of data. the second suggestion is probably less confusing I guess, but up to you whether you want to change it.